### PR TITLE
Add a script command for searching in Kinopoisk

### PR DESCRIPTION
**Pull request by andreiborisov**
## Description

This PR adds a new script command for searching in [Kinopoisk.ru](https://www.kinopoisk.ru) (a Russian analog of IMDB).

## Type of change

- [x] New script command

## Screenshot

CleanShot 2021-01-29 at 16 21 23@2x

## Dependencies / Requirements

No dependencies required.

## Checklist

- [x] I have read [Contribution Guidelines](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: commands/web-searches/kinopoisk.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title Search in Kinopoisk
+# @raycast.mode silent
+# @raycast.packageName Web Searches
+
+# Optional parameters:
+# @raycast.icon images/kinopoisk.png
+# @raycast.argument1 { "type": "text", "placeholder": "query", "percentEncoded": true }
+
```
Comment (unnamedd) [inline commands/web-searches/kinopoisk.sh:12]:
```suggestion

# Documentation
# @raycast.author Andrei Borisov
# @raycast.authorURL https://github.com/andreiborisov (or any social media, site you prefer to add)
# @raycast.description <A brief description about your Script Command>

```

**Review by unnamedd — CHANGES_REQUESTED** [review]
Hi @andreiborisov, thanks for your contribution. 

The Script Command looks good to me, you just forgot to add some meta tags to make it better in our auto-generated documentation!

File: commands/web-searches/kinopoisk.sh (hunk 1 above)
Comment (andreiborisov) [reply commands/web-searches/kinopoisk.sh:12]: Thanks!

File: commands/web-searches/kinopoisk.sh [hunk 2]
```diff
@@ -10,4 +10,8 @@
 # @raycast.icon images/kinopoisk.png
 # @raycast.argument1 { "type": "text", "placeholder": "query", "percentEncoded": true }
 
+# Documentation
+# @raycast.author Andrei Borisov
+# @raycast.authorURL https://github.com/andreiborisov (or any social media, site you prefer to add)
```
Comment (unnamedd) [inline commands/web-searches/kinopoisk.sh:15]: Don't forget to remove my instruction between the parenthesis! 🤣

```suggestion
# @raycast.authorURL https://github.com/andreiborisov
```

**Review by unnamedd — APPROVED** [review]

**Pull request merged**